### PR TITLE
Make the wait-for-port timeout adjustable

### PR DESCRIPTION
**Pull request by dshmatov**
**Description of the change**

Added the `wait-for-port` timeout and made it adjustable.

**Benefits**

Sometimes it is mandatory to set this timeout, especially when the huge amount of data is already in the influx and it tries to start but fails due to default 30s timeout. Therefore it's pretty reasonable to make it adjustable.

**Possible drawbacks**

Don't see any

**Applicable issues**

- https://github.com/bitnami/charts/issues/5182
- https://github.com/influxdata/influxdb/issues/19849
- https://github.com/bitnami/charts/issues/5519

### CODE REVIEW

File: 1/debian-10/rootfs/opt/bitnami/scripts/libinfluxdb.sh [hunk 1]
```diff
@@ -411,8 +412,8 @@ influxdb_start_bg_noauth() {
     INFLUXDB_HTTP_HTTPS_ENABLED=false INFLUXDB_HTTP_BIND_ADDRESS="127.0.0.1:${INFLUXDB_HTTP_PORT_NUMBER}" debug_execute "${start_command[@]}" &
 
     # branch 2 does not lisent on this port
-    [[ "${branch}" = "1" ]] && wait-for-port "$INFLUXDB_PORT_NUMBER"
-    wait-for-port "$INFLUXDB_HTTP_PORT_NUMBER"
+    [[ "${branch}" = "1" ]] && wait-for-port --timeout=$INFLUXDB_PORT_READINESS_TIMEOUT "$INFLUXDB_PORT_NUMBER"
```
Comment (miguelaeh) [inline 1/debian-10/rootfs/opt/bitnami/scripts/libinfluxdb.sh:415]:
```suggestion
    [[ "${branch}" = "1" ]] && wait-for-port --timeout="$INFLUXDB_PORT_READINESS_TIMEOUT" "$INFLUXDB_PORT_NUMBER"
```

Comment (miguelaeh) [reply 1/debian-10/rootfs/opt/bitnami/scripts/libinfluxdb.sh:415]: Please apply this change to all the uses of the variable

**Review by miguelaeh — CHANGES_REQUESTED** [review]
tank you for the PR please take a look to the suggested changes

**Comment by dshmatov** [comment]
@miguelaeh please check one, added quotes

File: 1/debian-10/rootfs/opt/bitnami/scripts/libinfluxdb.sh (hunk 1 above)
Comment (dshmatov) [reply 1/debian-10/rootfs/opt/bitnami/scripts/libinfluxdb.sh]: Ok, I'll do

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**